src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -134,6 +154,12 @@ TEST_F(Cgroups2Test, ROOT_CGROUPS2_AssignProcesses)
   AWAIT_EXPECT_WTERMSIG_EQ(SIGKILL, process::reap(pid));
 }
 
+
+TEST_F(Cgroups2CpuTest, ROOT_CGROUPS2_CpuStats)
+{
+  ASSERT_SOME(cgroups2::cpu::stats(TEST_CGROUP));
+}
+
```

Comment:
Instead of adding a new fixture, we can keep the same fixture but add a function that:
* enables requested controllers
* remembers the side effect
* disables the controllers that were added, during TearDown